slack_dump/slack_dump.py

```diff
@@ -29,28 +29,26 @@ def get_real_time_message(token, start1):
         while not slack.rtm_connect():
             time.sleep(CONNECTION_SLEEP_INTERVAL)
         end = format(time.time(), '0.6f')
-
         slack_history.get_all_channels_history(token,start1,end)
 
         while True:
             msg = slack.rtm_read()
             if not msg:
                 time.sleep(EMPTY_READ_SLEEP_INTERVAL)
                 continue
-	    print (msg)
+	    print msg
 
             if not len(msg[0]) == 1:
                 try:
                     key = hashlib.sha1(msg[0]['channel'] + msg[0]['ts']).hexdigest()
 		    msg[0]["key"] = key
 		    res=db.slack_db.insert_one(msg[0])
 		    print "Inserted!!"
-
-		except Exception:
+		            except Exception:
```

Comment:
I think there an indentation problem here. Could check of the script is running.